CHANGELOG.md

```diff
@@ -4,6 +4,51 @@ All notable changes to Zebra will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-alpha.13](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-alpha.13) - TODO: Update with release date before merging
+
+### Added
+
+- Reject duplicate sprout nullifiers in the state (#2477)
+- Use the block verifier and non-finalized state in the cached state tests (#2463)
```

Comment:
I think this might be a test coverage fix?
(We claimed the integration tests were testing the block verifier and non-finalized state, but they actually weren't.)

Reply:
Moved it to `Fixed` and reworded it to explain what the impacts were.